docs/roadmap.rst

```diff
@@ -45,14 +34,11 @@ KPIs
 Features
 ++++++++
 
--   process overview can be filtered by region/city
 -   idea collection process with or without a map
 -   private and public processes
--   project managers can invite new users
+-   new users can be invited by liqd
```

Comment:
I would avoid mentioning liqd here because it is the main contributor of adhocracy, but not the only one (hopefully)